action_list_sample/panel.js

```diff
@@ -34,35 +63,37 @@ export default class Panel extends Component{
             icon = this.icons['up'];  
         }
         
-        return ( 
+        return (
+            <Animated.View 
+            style={[styles.container,{height: this.state.animation}]}>
             <View style={styles.container} >
-                <View style={styles.titleContainer}>
+                <View style={styles.titleContainer} onLayout={this._setMinHeight.bind(this)}>
```

Comment:
Don't bind in a property or you'll be creating a new function every time the state/props change.